### ISSUE TITLE
Lookup protocol data at top-level of module, without specifying a dataset version?

### ISSUE THREAD

**Issue by deathcap** [PrismarineJS/node-minecraft-data#14]
The usage for looking up protocolVersions with node-minecraft-data seems to be slightly awkward, requiring calling the module with a supported version:

```javascript
> require('minecraft-data')('1.8.9').versionsByMinecraftVersion['1.5.2']
{ minecraftVersion: '1.5.2',
  version: 61,
  usesNetty: false,
  majorVersion: '1.5' }
```

even though this is the same data in all datasets:

```
> require('minecraft-data')('1.9').versionsByMinecraftVersion['1.5.2']
{ minecraftVersion: '1.5.2',
  version: 61,
  usesNetty: false,
  majorVersion: '1.5' }
```

and you have to know a current dataset to lookup any version:

```
> require('minecraft-data')('1.5.2').versionsByMinecraftVersion['1.5.2']
TypeError: Cannot read property 'versionsByMinecraftVersion' of null
```

Wouldn't it be cleaner if you could lookup `versions`, and `versionsByMinecraftVersion` like this:

```javascript
> require('minecraft-data').versionsByMinecraftVersion['1.5.2']
```

or perhaps to maintain the function-calling syntax, something like:

```javascript
> require('minecraft-data')('common').versionsByMinecraftVersion['1.5.2']
```

(I started to make this change myself, but got stuck on building the indexes)

**Comment by rom1504** [comment]
Yeah I guess `require('minecraft-data').versionsByMinecraftVersion['1.5.2']` is indeed better.
It's possible by "abusing" the fact js functions are objects.
I'll do that.

**Comment by rom1504** [comment]
Released as 0.19.0
